src/Breadcrumbs/index.js

```diff
@@ -23,15 +25,32 @@ const StyledBreadcrumbsList = styled.ol`
   padding: 0;
 `;
 
-const Breadcrumbs = ({ children, dataTest }: Props) => (
+const StyledBackButton = styled(Button)`
```

Comment:
Even that this is possible because we use rest props inside `Button` component, I would not extend it as we don't provide this option officially.

Reply:
@vepor what would you recommend as a best approach? Wrapping that inside new inline styled component, or using Stack?